From 48648570e449cf7f84a26dc03c1e3f620fa69757 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Mon, 29 Feb 2016 18:13:39 +0200
Subject: [PATCH] OptionsGTK.cmake: drop the hardcoded introspection/gtkdoc
 disabling when cross-compiling

This was not possible to override from the command line and in OpenEmbedded
(one of the most prominent cross-compilation frameworks) introspection does work fine,
through the use of qemu target emulation.

Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>

Upstream-Status: Pending
---
 Source/cmake/OptionsGTK.cmake | 6 ------
 1 file changed, 6 deletions(-)

Index: webkitgtk-2.12.1/Source/cmake/OptionsGTK.cmake
===================================================================
--- webkitgtk-2.12.1.orig/Source/cmake/OptionsGTK.cmake
+++ webkitgtk-2.12.1/Source/cmake/OptionsGTK.cmake
@@ -424,12 +424,6 @@ if (USE_LIBHYPHEN)
     endif ()
 endif ()
 
-# Override the cached variables, gtk-doc and gobject-introspection do not really work when cross-building.
-if (CMAKE_CROSSCOMPILING)
-    set(ENABLE_GTKDOC OFF)
-    set(ENABLE_INTROSPECTION OFF)
-endif ()
-
 # Override the cached variable, gtk-doc does not really work when building on Mac.
 if (APPLE)
     set(ENABLE_GTKDOC OFF)
